Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -95,14 +95,18 @@ def test_module():
     """
     Sends a test message to the dedicated slack channel.
     """
-    if not DEDICATED_CHANNEL:
-        return_error('A dedicated slack channel must be provided.')
-    channel = get_conversation_by_name(DEDICATED_CHANNEL)
-    if not channel:
-        return_error('Dedicated channel not found.')
-    message = 'Hi there! This is a test message.'
+    if not DEDICATED_CHANNEL and len(PERMITTED_NOTIFICATION_TYPES) > 0:
+        return_error(
+            "When 'Types of Notifications to Send' is populated, a dedicated channel is required.")
```

Comment:
did you see this happen or this is "a just in case" check?

Reply:
It happens. If someone wants to have a couple notification types, but doesn't specify the channel (since it's no longer required) then the test button should catch it.